ansible/library/announce_routes.py

```diff
@@ -206,8 +206,8 @@ def generate_routes(family, podset_number, tor_number, tor_subnet_number,
                     # Skip non podset 0 for T0
                     if podset != 0:
                         continue
-                    # Skip subnet 0 (vlan ip) for M0
-                    elif topo == "m0" and subnet == 0:
+                    # Skip subnet 0 (vlan ip)
+                    elif subnet == 0:
```

Comment:
I doubting this change will cause T1 topo miss first vlan route.

Reply:
Should we only add the condition for m0 and t0 topo? Only them have directly VLAN and require to skip DUT's vlan. For t1 topo, we shouldn't skip this subnet.

Reply:
This is under route type == 'ToR"? (line 205), is it still a concern?

Reply:
The router type here is DUT's neighbors' type. For T1 DUT, it has tor neighbors and will go this branch.